bin/rad-project

```diff
@@ -56,22 +56,22 @@
     (def name (lookup :name (get-meta! project)))
     (def origin (lookup :id (first-rsm-of-type! project :rad-repo)))
     (match (process! "test" ["-d" name] "")
-      :ok (do
-            (put-str! (string-append
-                        "The directory \"" name "\" already exists.\n"
-                        "Refusing to continue."))
-            (exit! 1))
-      _   (put-str! name))
+           :ok (do
+                 (put-str! (string-append
+                            "The directory \"" name "\" already exists.\n"
+                            "Refusing to continue."))
+                 (exit! 1))
+           _   (put-str! name))
```

Comment:
These whitespace changes are a bit confusing. We should avoid them in the future if possible to keep the review size small

Reply:
Well, I think we should rather have a standard formatting.